res_company_active/demo/res_company.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
```

Comment:
Usually demo data are with noupdate="1", isn't it better?
Not blocking from my part though.

Reply:
I agree with that and I made the change, thanks !